packages/request-client.js/test/index.test.ts

```diff
@@ -281,16 +289,19 @@ describe('index', () => {
     mock.onGet('/getTransactionsByChannelId').reply(200, {
       result: { transactions: [TestData.timestampedTransactionWithoutExtensionsData] },
     });
+    mock.onGet('/getConfirmedTransaction').reply(200, { result: {} });
 
     const requestNetwork = new RequestNetwork({
       nodeConnectionConfig: { baseURL },
       signatureProvider: fakeSignatureProvider,
     });
-    await requestNetwork.createRequest({
+    const request = await requestNetwork.createRequest({
       requestInfo: TestData.parametersWithoutExtensionsData,
       signer: payeeIdentity,
     });
     expect(spy).to.have.been.called.once;
+
+    await new Promise((resolve): any => request.on('confirmed', resolve));
```

Comment:
this is repeated a lot, maybe add a new method?
```typescript
public waitForConfirmation(): Promise<IRequestData>{
    return new Promise(resolve => this.on('confirmed', resolve);
}
```
usage: `await request.waitForConfirmation()`